perl/p5-tcl-ptk/Portfile

```diff
@@ -25,25 +25,29 @@ long_description    Tcl::pTk interfaces perl to an existing Tcl/Tk \
                     features (for example native-looking widgets using the \
                     Tile package).\n
 
-checksums           rmd160  a184dd74f2c135d16737e899b93eb08abae7f87b \
-                    sha256  64f5089147e1700f2f70051ea2aa9dcc4896f68659b8dd986585f9febb3daa1a \
-                    size    538814
+checksums           rmd160  7c5fa3d31d11d91b9ccbbdd3757a130fbdf0bd8f \
+                    sha256  4e88b6280f35da97b216f2d507c7a89c54d11e2097f4868509049de5b845adc1 \
+                    size    537003
 
 if {${perl5.major} != ""} {
     # Only used for a test removed by patch;
     # doesn't get stored in configuration
     #configure.args-append \
     #                --tclsh ${prefix}/bin/tclsh
 
-    # TODO: if and when Tix and Tktable ports support aqua,
-    # add those as dependencies
     depends_lib-append \
                     port:tk \
+                    port:tklib \
+                    port:tkimg \
+                    port:tix \
+                    port:tktable \
```

Comment:
Would you please sort the dependencies? Not just the new ones, but all of them.

Reply:
Done in f82d4ab2a9